src/groupeddatatable/grouping.jl

```diff
@@ -193,7 +193,7 @@ combine(map(d -> mean(dropnull(d[:c])), gd))
 """
 function combine(ga::GroupApplied)
     gd, vals = ga.gd, ga.vals
-    valscat = vcat(vals)
+    valscat = vcat(vals...)
```

Comment:
vcat used to wrap everything into a Vector{DataTables} https://github.com/JuliaData/DataTables.jl/blob/master/src/abstractdatatable/abstractdatatable.jl#L714. I tried to make this behave more like Base.vcat